build-tools/packages/build-cli/src/commands/generate/typetests.ts

```diff
@@ -190,8 +191,15 @@ declare type MakeUnusedImportErrorsGoAway<T> = TypeOnly<T> | MinimalType<T> | Fu
 
 		mkdirSync(outDir, { recursive: true });
```

Comment:
option: make this async now too.

Reply:
Replaced this and rmSync with their async counterparts in latest.